### ISSUE TITLE
Node Security Project Vulnerabilities

### ISSUE THREAD

**Issue by tjsail33** [karma-runner/karma#2529]
### Expected behaviour
```bash
nsp check
```
should report no security vulnerabilities.

### Actual behaviour
```
┌───────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│               │ DoS due to excessively large websocket message                                                                                               │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Name          │ ws                                                                                                                                           │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Installed     │ 1.0.1                                                                                                                                        │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Vulnerable    │ <=1.1.0                                                                                                                                      │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Patched       │ >=1.1.1                                                                                                                                      │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Path          │ my-package@3.7.1 > karma@1.3.0 > socket.io@1.4.7 > engine.io@1.6.10 > ws@1.0.1                                                                │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/120                                                                                                       │
└───────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                                                                                                         │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Name          │ negotiator                                                                                                                                   │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Installed     │ 0.4.9                                                                                                                                        │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Vulnerable    │ <= 0.6.0                                                                                                                                     │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Patched       │ >= 0.6.1                                                                                                                                     │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Path          │ my-package@3.7.1 > karma@1.3.0 > socket.io@1.4.7 > engine.io@1.6.10 > accepts@1.1.4 > negotiator@0.4.9                                        │
├───────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/106                                                                                                       │
└───────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

### Environment Details

- Karma version (output of `karma --version`): 1.3.0

### Steps to reproduce the behaviour

1. `npm install -g nsp`
2. `nsp check`

### Comments
I noticed that the package.json in `master` has already bumped the affected dependency, I'm mostly hoping that this can be packaged up into a release and tagged. I'd like to add security detection to our nightly build, and this is a blocker for doing so.

**Comment by dignifiedquire** [comment]
Thanks for the report, will publish a new version soon.

**Comment by tjsail33** [comment]
thanks so much @dignifiedquire!

**Comment by aqrln** [comment]
@dignifiedquire any update on this?

**Comment by dignifiedquire** [comment]
Just published 1.4 🎉 https://github.com/karma-runner/karma/releases/tag/v1.4.0
